CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ set(CMAKE_C_VISIBILITY_PRESET hidden)
 set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 set(CMAKE_VISIBILITY_INLINES_HIDDEN ON)
 
+# Fixes warning ld: warning: ignoring duplicate libraries on Mac and Windows.
+CMAKE_POLICY(SET CMP0156 NEW)
```

Comment:
This needs `if (POLICY CMP0156)` because there's `cmake_minimum_required(VERSION 3.18)`.
Also please don't use SHOUTY syntax.